src/services/mockData.ts

```diff
@@ -14,6 +15,41 @@ const notifications: Notification[] = [
   }
 ]
 
+const auditTrail: AuditTrailRecord[] = [
+  {
+    process: 'Create user',
+    actionTaken: 'newUserCreated',
+    links: '',
+    when: '10/09/2022 8:16',
+    changedBy: 'User Id 12345',
+    comment: ''
+  },
+  {
+    process: 'Auto-update golden record field',
+    actionTaken: 'grUpdate',
+    links: 'Golden ID (G543)',
+    when: '11/09/2022 10:07',
+    changedBy: 'System',
+    comment: 'Golden ID (G543) updated field values (name, cell number)'
+  },
+  {
+    process: 'Autolink',
+    actionTaken: 'grAndPatientRecordLinked',
+    links: `Golden ID (G123),Golden ID (P123-1)`,
+    when: '12/09/2022 9:00',
+    changedBy: 'System',
+    comment: 'Golden ID (G123) linked with Patient ID (P123-1)'
+  },
+  {
+    process: 'Approve link',
+    actionTaken: 'grAndPatientRecordLinkApproved',
+    links: 'Golden ID (G127),Golden ID (P127-1)',
```

Comment:
Following design, these should be clickable links, so maybe we should have array of  IDs  instead ? So that you could generate a link to the patient details page.